serde-generate/src/swift.rs

```diff
@@ -91,6 +91,23 @@ impl<'a> CodeGenerator<'a> {
 
         Ok(())
     }
+
+    /// Whether case convention matters or not.
+    ///
+    /// If `true`, transform following the [Swift case conventions](https://www.swift.org/documentation/api-design-guidelines/#conventions).
```

Comment:
Good catch. We already auto-fix the case in several languages including Swift. Can you just call `.to_mixed_case()` in the few places where it's needed? (No need for a config option or even a method)

Reply:
This looks good to me, because it's the first time to generate code for my use case.
My main concern here is naming change in the generated code could break existing usage. If it's acceptable to proceed in this manner, I'll keep the changes straightforward.

Reply:
Yes, that is generally a good remark but in this case, I feel that users can upgrade to the new version of `serde-generate` in their own time.

Reply:
Make sense. Updated. 🔧